pages/1.10/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -8,26 +8,322 @@ excerpt:
+
+[This section](#example-use-cases) shows how to set these configuration parameters.
+
+# Configuration parameter reference
+## ca\_certificate\_path
+Path to a file within the `genconf/` directory containing a single X.509 CA certificate in the OpenSSL PEM format. For example: `genconf/CA_cert`. It is either a *root CA certificate* (“self-signed”) or an *intermediate CA certificate* (“cross-certificate”) signed by some other certificate authority.
+
+If provided, this is the custom CA certificate. It is used as the signing CA certificate, i.e., the DC/OS CA will use this certificate for signing end-entity certificates (the subject of this certificate will be the issuer for certificates signed by the DC/OS CA). If not provided, the DC/OS cluster generates a unique root CA certificate during the initial bootstrap phase and uses that as the signing CA certificate.
+
+The public key associated with the custom CA certificate has to be of type RSA.
+
+## ca\_certificate\_key\_path
+Path to a file within the `genconf/` directory containing the private key corresponding to the custom CA certificate, encoded in the OpenSSL (PKCS#8) PEM format. For example: `genconf/CA_cert.key`.
+
+**Note**: this is highly sensitive data. The configuration processor accesses this file only for configuration validation purposes, and does not copy the data. After successful configuration validation this file needs to be placed out-of-band into the file system of all DC/OS master nodes to the path /var/lib/dcos/pki/tls/CA/private/custom_ca.key before most DC/OS systemd units start up. The file has to be readable by the root user, and should have have 0600 permissions set.
+
+Required if `ca_certificate_path` is specified.
+
+## ca\_certificate\_chain\_path
+Path to a file within the `genconf/` directory containing the complete CA certification chain required for end-entity certificate verification, in the OpenSSL PEM format. For example: genconf/CA_cert_chain.pem.
+Has to be left undefined if `ca_certificate_path` points to a root CA certificate.
+Required if ca_certificate_path is specified and if the custom CA certificate is an *intermediate CA certificate*. 
+
+This needs to contain all CA certificates comprising the complete sequence starting precisely with the CA certificate that was used to sign the custom CA certificate and ending with a root CA certificate (where issuer and subject are equivalent), yielding a gapless certification path. The order is significant and the list has to contain at least one certificate.
+
+
+# Installing DC/OS Enterprise with a custom CA certificate
+## Starting point
+Based on the requirements described above, this is the starting point for the installation:
+
+- The installation of DC/OS Enterprise via the Advanced Installer has been prepared according to the corresponding [documentation](/1.10/installing/ent/custom/advanced/).
```

Comment:
It appears that the following steps must be done between the Configure and Install steps of the linked page. It might help to be more specific about where to stop in the Advanced installation.